packages/notification/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@kaizen/notification",
+  "version": "0.0.0",
```

Comment:
Does this auto update from 0.0.0?

Reply:
Yeah I'm hoping so, because I've got `BREAKING CHANGE` on this. Usually it just goes from 1.0.0 -> 1.1.0 without the `BREAKING CHANGE` part.

Not the end of the world if it doesn't work though.